### PR TITLE
Update readme with instruction to generate  TLS

### LINKED ISSUE #15: Instructions to create and use your own TLS certificate for local development
In https://github.com/georchestra/docker/tree/master/resources/ssl, certificates are provided to allow dockerized applications to use HTTPS without any problem.
But I understand it's a security problem for the developer, because if they trust the certificate, as the private key is published on the web, any website could use it, and your browser would trust it. See https://letsencrypt.org/docs/certificates-for-localhost/. (I understand that the attacker should also realize MITM on DNS lookup, but it's still possible, since most of the DNS traffic is not encrypted nowadays...)

For this reason, I understand that it would be safer for the developer to generate its own certificate, and to tell the dockerized applications to use it.

### PR DESCRIPTION
**Pull request by RemiDesgrange**
Fix #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,18 @@ Stop geOrchestra with
 docker-compose down
 ```
 
+## SSL/TLS
+
+This repo comes with a self signe cert which is _not_ valid. To test with a valid cert you can
```
Comment (fvanderbiest) [inline README.md]: signed

File: README.md [hunk 2]
```diff
@@ -44,6 +44,18 @@ Stop geOrchestra with
...
+install [mkcert](https://github.com/FiloSottile/mkcert) and do the following:
```
Comment (fvanderbiest) [inline README.md]: where should it be installed ? Sorry if it sounds obvious ;-)

File: README.md [hunk 3]
```diff
@@ -44,6 +44,18 @@ Stop geOrchestra with
...
+
+* `mkcert -install`. Only do this once ! It'll install a fake root cert in system store (and some
+  other see mkcert doc)
```
Comment (fvanderbiest) [inline README.md]: some others ?

**Review by fvanderbiest — CHANGES_REQUESTED** [review]
Thanks !

File: README.md (hunk 3 above)
Comment (RemiDesgrange) [reply README.md]: Firefox  (and maybe Chrome, but not and expert). But I'd like to be as concise here as possible and if people are interested, they'll check the doc.

File: README.md (hunk 2 above)
Comment (RemiDesgrange) [reply README.md]: you mean `mkcert` ? I though that the person who wants to use this will find his distro/OS and install it the way they want.

Comment (fvanderbiest) [reply README.md]: Yes, so, I would say "you can
install mkcert on your host and do the following:"

File: README.md (hunk 3 above)
Comment (fvanderbiest) [reply README.md]: I meant "some others", not "some other", for the typo ;-)

Comment (RemiDesgrange) [reply README.md]: :sweat_smile:

**Comment by fvanderbiest** [comment]
Thanks Remi.
Do you think we could make the Traefik service depend on another service based on mkcert (eg https://hub.docker.com/r/vishnunair/docker-mkcert) to generate a new cert before the LB starts ?

**Comment by RemiDesgrange** [comment]
Hum, I'd rather not because mkcert (or any other solution) imply that we have to install the root cert on the user host (somewhere) I find it a bit intrusive.

**Comment by fvanderbiest** [comment]
Many software store things on `~/.something`. I would not be ashamed ;-)

**Pull request merged**